systemtests/bin/setup_download_tests.py

```diff
@@ -36,13 +38,27 @@
 
 
 def iterate_through_symbols_files(
+    baseurl,
     auth_token,
-    sym_files_url,
     start_page,
+    try_symbols,
     sym_file_type_to_filename,
     end_condition_num_files,
 ):
-    sym_files_generator = get_sym_files(auth_token, sym_files_url, start_page)
+    params = {
+        "page": start_page,
+        "size": "< 10mb",
+        # NOTE(willkg): this restricts the search to symbols files only; when we decide
+        # to download other kinds of files, we'll need to remove this
+        "key": [".sym"],
+    }
+    if try_symbols:
+        params["key"].append("try/")
```

Comment:
This filters on files that have `.sym` and files that have `try/`. That helps for when we're looking for try symbols since it reduces the results to a set closer to our needs.

However, there's no way to say "not try symbols", so it's not helpful for finding regular symbols. :cry: